### PR TITLE
[nearstudio] add quickstart mode as a url param. close the tree view …

### PR DESCRIPTION
**Pull request by janedegtiareva**
…on this parameter. 

(needs unit tests which will be made in a follow-up commit)

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -242,6 +253,9 @@ export class App extends React.Component<AppProps, AppState> {
       if (project.getFile("README.md")) {
         openFiles([["README.md"]]);
       }
+      if (this.state.quickStart) {
+        openFiles([["assembly/main.ts", "src/main.js"]]);
```
Comment (vgrichina) [inline src/components/App.tsx]: let's store in fiddle which files need to be open, especially given `fiddle` is available here

File: src/components/App.tsx [hunk 2]
```diff
@@ -406,25 +420,29 @@ export class App extends React.Component<AppProps, AppState> {
     return this.state.hasStatus;
   }
 
+  toggleWorspaceSplit() {
+    const workspaceSplits = this.state.workspaceSplits;
```
Comment (vgrichina) [inline src/components/App.tsx]: generally I prefer when state change methods are idempotent when possible. Let's in follow-up changes make separate methods to create split / restore single pane?

File: src/components/App.tsx [hunk 3]
```diff
@@ -406,25 +420,29 @@ export class App extends React.Component<AppProps, AppState> {
     return this.state.hasStatus;
   }
 
+  toggleWorspaceSplit() {
```
Comment (vgrichina) [inline src/components/App.tsx]: Wor_k_space

File: src/index.tsx [hunk 4]
```diff
@@ -114,7 +115,13 @@ export async function init(environment = "production") {
     } else {
       ReactDOM.render(
         <ErrorBoundary>
-          <App update={update} fiddle={fiddle} embeddingParams={embeddingParams} windowContext={appWindowContext}/>
+          <App
+            update={update}
+            fiddle={fiddle}
+            embeddingParams={embeddingParams}
+            windowContext={appWindowContext}
+            quickStart={quickStart}
```
Comment (vgrichina) [inline src/index.tsx]: Should `quickStart` be part of `windowContext` or `embeddingParams`?

Comment (janedegtiareva) [reply src/index.tsx]: I prefer a top level parameter to make the url look as readable as possible

File: src/components/App.tsx (hunk 3 above)
Comment (janedegtiareva) [reply src/components/App.tsx]: thanks, fixed.

File: src/components/App.tsx (hunk 2 above)
Comment (janedegtiareva) [reply src/components/App.tsx]: added a todo

File: src/components/App.tsx (hunk 1 above)
Comment (janedegtiareva) [reply src/components/App.tsx]: Adding a todo to address in a diff pull request. I think that this is a good enough first stab.

Comment (vgrichina) [reply src/components/App.tsx]: Let's try to put things into https://github.com/nearprotocol/NEARStudio/issues and move into appropriate milestones.

**Pull request merged**